src/components/categoricaldisplay/FilterPanel.tsx

```diff
@@ -42,6 +42,48 @@ const FilterItem = React.memo(
     }
 );
 
+const FilterGroup = React.memo(
+    ({ focusedCategoryKey, activeCategoryKey, handleFilterClick, category, isSearching }: any) => {
+        const [isGroupActive, setIsGroupActive] = React.useState(false);
+        const [isOpen, setIsOpen] = React.useState(false);
+
+        return (
+            <>
+                <div
+                    className={`sc-mcd__filter__item ${isGroupActive && !isSearching ? 'sc-mcd__filter__item--selected' : ''}`}
+                    onClick={() => setIsOpen(!isOpen)}
+                >
+                    <FilterItemIcon categoryId={category.categoryId} />
+                    {t.translate(category.categoryName)}
```

Comment:
Translate won't work this way. It will only work for string literals. If the `categoryName` is from the API then it will be translated in the BE.

Reply:
Right, I removed it👍🏻